src/network_analysis.jl

```diff
@@ -122,6 +136,7 @@ function reactioncomplexes(::Type{SparseMatrixCSC{Int, Int}}, rn::ReactionSystem
     complexes, B
```

Comment:
Here, and in a lot more cases, I wonder if there are more descriptive names for the matrices? I am really not familiar with standards for network analysis though, and I only think it happened like once or twice that I had to do a actual retake to go around to look what kind of matrix I was dealing with. 
(I have no strong opinions, but a general thought I had)

Reply:
Maybe we could go through and use incidencemat, complexstoichmat, netstoichmat, etc. for these variable names, or maybe standardize the letters we use for each of the matrices and have that somewhere in the documentation (I believe the most common I have seen are Y or Z for the complex stoichiometry matrix, D for the incidence matrix, and S or \Gamma for the reaction stoichiometry matrix)

Reply:
Except you don't want a variable name to be the same as function names we provide so they should be something different.

Reply:
Perhaps we should have made the functions `get_netstoichmat` and such but it is too late now for that as it would be breaking. (And our naming is not inconsistent with how base Julia names many functions.)